tests/crypto/enclave/enc/enc.c

```diff
@@ -145,64 +147,25 @@ OE_DEFINE_SYSCALL1_M(SYS_close)
     OE_TEST(OE_OK == f_close(&rval, (int)arg1));
     return rval;
 }
-
-static long _syscall_dispatch(
-    long number,
-    long arg1,
-    long arg2,
-    long arg3,
-    long arg4,
-    long arg5,
-    long arg6)
-{
-    OE_UNUSED(arg5);
-    OE_UNUSED(arg6);
-
-    switch (number)
-    {
-        OE_SYSCALL_DISPATCH(SYS_openat, arg1, arg2, arg3, arg4);
-#if defined(__x86_64__) || defined(_M_X64)
-        OE_SYSCALL_DISPATCH(SYS_open, arg1, arg2, arg3);
 #endif
-        OE_SYSCALL_DISPATCH(SYS_read, arg1, arg2, arg3);
-        OE_SYSCALL_DISPATCH(SYS_readv, arg1, arg2, arg3);
-        OE_SYSCALL_DISPATCH(SYS_close, arg1);
-        default:
-            return -1;
-    }
-}
 
-static oe_result_t _syscall_hook(
-    long number,
-    long arg1,
-    long arg2,
-    long arg3,
-    long arg4,
-    long arg5,
-    long arg6,
-    long* ret)
+void test(char* abspath)
 {
-    oe_result_t result = OE_UNEXPECTED;
-    if (ret)
-        *ret = -1;
-
-    if (!ret)
-        OE_RAISE(OE_INVALID_PARAMETER);
+#if !defined(CODE_COVERAGE) && defined(OE_USE_OPENSSL)
+    /*
+     * When enabling code coverage analysis, libgcov should initialize the host
+     * fs already so we do not do it again here. Otherwise, the
+     * oe_load_module_host_file_system will fail.
+     */
+    if (oe_load_module_host_file_system() != OE_OK)
```

Comment:
@mingweishih  @justanotherminh is this correct? If CODE_COVERAGE is not defined, the file system module is loaded. It should be if CODE_COVERAGE is defined.

Reply:
This is correct. When CODE_COVERAGE is enabled, the `libgcov` will do this for us. When CODE_COVERAGE is not enabled, we will need to make this call to opt into the host fs.